resources/test_data/tbl/tpch/sf-0.01/nation.tbl

```diff
@@ -0,0 +1,27 @@
+n_nationkey|n_name|n_regionkey|n_comment
```

Comment:
I feel bad about this. If we keep the tests they are right now and want to continue verifying a against a tbl version that came from dbgen, we need scale factors that are actually legal. SF 0.001 (used before) was not.